src/unaudited/ECDSAStakeRegistryStorage.sol

```diff
@@ -27,14 +29,19 @@ abstract contract ECDSAStakeRegistryStorage is ECDSAStakeRegistryEventsAndErrors
     /// @notice Defines the duration after which the stake's weight expires.
     uint256 internal _stakeExpiry;
 
+    /// @notice Maps an operator to their signing key history using checkpoints
+    mapping(address => CheckpointsUpgradeable.History)
+        internal _operatorSigningKeyHistory;
+
```

Comment:
do you care at all about storage compatibility for upgrades? or not concerned yet because we can simply redeploy instead of upgrading

Reply:
Yeah not concerned about it rn, once we have people using it we can be cautious about these changes

Reply:
cool, works for me. I'll resolve this thread then.